run.sh

```diff
@@ -54,6 +54,12 @@ function terminate()
     cd ..
 }
 
+# If IFACE_NAME is empty, read the Wi-Fi interface name from sec.conf
+if [[ -z $IFACE_NAME ]]; then
+    IFACE_NAME="$(head -n 1 ./wpa_supplicant/sec.conf)" | cut -d : -f 2 # get the second slice (colon separated) of the first line of the file
```

Comment:
IFACE_NAME="$(head -n 1 ./wpa_supplicant/sec.conf | cut -d : -f 2)" # get the second slice (colon separated) of the first line of the file